cves/kernel/CVE-2019-20054.yml

```diff
@@ -343,10 +274,54 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+description: "sysctl is used for reading and writing attributes to the Linux
+  Kernel. It is both a system call and a command to be used in a terminal."
```

Comment:
This description doesn't seem to say much about the vulnerability, it is just a definition of sysctl.